1.6/run.sh

```diff
@@ -13,15 +13,15 @@ unzip -q /usr/lib/jenkins/jenkins.war -d /tmp/war
 new_password_hash=`obfuscate_password ${JENKINS_PASSWORD:-password}`
 
 if [ ! -e /var/lib/jenkins/configured ]; then
-  cp -r /opt/openshift/configuration/* /var/lib/jenkins
+  echo "[INFO] Installing Jenkins configuration and plugins to ${JENKINS_HOME} ..."
+  cp -r /opt/openshift/configuration/* ${JENKINS_HOME}
   rm -rf /opt/openshift/configuration/*
-  
+
+  echo "[INFO] Creating initial 'admin' user with password 'password'"
```

Comment:
it's not necessarily "password" and i don't think you want to print the actual password, so maybe just "creating initial admin user"